gen_cert.py

```diff
@@ -85,14 +85,18 @@ def get_cert_date(
     - If using the "ROLLING" certs feature, use today's date
     - If all else fails use 'configured_date_parameter' for date
     """
-
+    utc_zone = get_timezone('UTC')
```

Comment:
this is just for the initial utc replacement, I set the actual timezone later...